AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLiteStorageEngineAdapterTests.swift

```diff
@@ -375,4 +375,120 @@ class SQLiteStorageEngineAdapterTests: BaseDataStoreTests {
             XCTFail(String(describing: error))
         }
     }
+
+    func testClearIfNewVersionShouldStoreInitialVersion() {
+
+        guard let userDefaults = UserDefaults.init(suiteName: "testVersionIsEmpty") else {
+            XCTFail("Could not create a UserDafult with this suite name")
+            return
+        }
+        userDefaults.removeObject(forKey: SQLiteStorageEngineAdapter.dbVersionKey)
+
+        let newVersion = "newVersion"
+
+        let mockFileManager = MockFileManager()
```

Comment:
ensure filemanager removeItem is not called,
one way to achieve this is 
```
fileManager.removeItem = {
 XCTFail("Should not have called removeItem")
}
```

Reply:
En, this is good